tests/test_save.py

```diff
@@ -0,0 +1,99 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+
+def test_save(tmpdir: "Path", testonto: "Ontology") -> None:
+    import pytest
+
+    from ontopy import get_ontology
+    from ontopy.ontology import EMMOntoPyException
+    from pathlib import Path
+
+    # Save ontology in a different location
+    testonto.save(tmpdir / "testonto_saved.ttl")
```

Comment:
I stedet for å skrive til en temporær mappe som automatisk slettes når testen har kjørt, er det enklere å debugge hvis man skriver til en output/ undermappe under tests/. Hvis du legger inn en .gitignore fil i den mappen som inneholder en linje med "*", vill den mappen bli en del av git-repoet men alle filene som blir generert i den vill bli ignorert av git. Men de vill fortsatt være enkelt tilgjengelige for debugging.

Reply:
As you wish... but I really don't see the point of such complexity. Why adding options to the testing? Tests are not distributed production code, but ment to be simple and helpful for ensuring code quality. Easy debugging is part of that.

All extra complexity added to the tests just make the code harder to maintain - and will sooner or later require that we have to add tests for the tests...